academic/R/R.info

```diff
@@ -1,10 +1,10 @@
 PRGNAM="R"
-VERSION="4.1.2"
+VERSION="4.1.3"
 HOMEPAGE="https://www.r-project.org/"
-DOWNLOAD="UNSUPPORTED"
-MD5SUM=""
-DOWNLOAD_x86_64="https://cran.r-project.org/src/base/R-4/R-4.1.2.tar.gz"
-MD5SUM_x86_64="6e28db9d02c6d3dae51a149b8e261ab1"
+DOWNLOAD="https://cran.r-project.org/src/base/R-4/R-4.1.3.tar.gz"
```

Comment:
This errors on i586 since there is hard check on line 66. Did did you mean to remove DOWNLOAD="UNSUPPORTED"?

Reply:
I see that error now. No, this because slackbuilds has r-studio legacy which exists only to support 32-bit. Thus lacking a 32-bit build of R makes the legacy studio pointless.

Reply:
Thanks, this built fine on 32bit now with #1489